.travis.yml

```diff
@@ -11,18 +23,21 @@ env:
     - secure: "OpBwPc1GNvauageYOH3RscAa7wpZxgpmqDz15aigIKLNWzAhAtVUx0MleZ8rQeoqml6nrAvlnzuVHjKL2lVcjMPpjUis7bcQ5UAGK7tZK8x+qZNQxXmpXu8+pENwQA2yFaqt/xy7K5jFOrHJHTRxcPnyVG1yKakPWz53PPYUwbc="
     # SONA_PASS
     - secure: "Xw7rI/qlML1nD2e2XwlakkhKAWNGZKqqE+Q3ntTvFpfHryl7KLCvVzJ4LIavnL6kGJaWOgy9vlSoEWn5g9nqHSfE31C/k5pY5nTMAKiwiJzfAS+r0asKXW2gmKhwtcTBkqyLVOZLCJSPVlFRQyfBJHY+Fs0L3KWcnMQgtBlyDhU="
+  matrix:
+    - SCALAJS_VERSION=
+    - SCALAJS_VERSION=0.6.23
+    - SCALAJS_VERSION=1.0.0-M3
 
-script: admin/build.sh
-
-addons:
-  apt:
-    packages:
-      - openjdk-6-jdk
+matrix:
+  exclude:
+    - scala: 2.13.0-M3
+      env: SCALAJS_VERSION=1.0.0-M3
+    - scala: 2.12.6
+      jdk: openjdk6
+    - scala: 2.13.0-M3
+      jdk: openjdk6
```

Comment:
There are 19 builds at the moment.  Do we really need all of them?  

Trimming down would be nice, because the wall time used to be 10 minutes for 3 concurrent builds.  The individuals builds are shorter, but now it looks like it's over 20 minutes because Travis throttles to 4 or 5 concurrent builds.

Reply:
Yeah, sounds like a good start.  

I'm thinking ahead to #110 when things will really go downhill.